internal/api/mfa_test.go

```diff
@@ -36,6 +36,12 @@ type MFATestSuite struct {
 	TestSession  *models.Session
 }
 
+type ErrorResponse struct {
+	Code    int    `json:"code"`
+	Message string `json:"msg"`
+	ErrorID string `json:"error_id"`
+}
```

Comment:
There's some struct in `errors.go` that can be reused, but as you wish.

Reply:
I missed that, thanks for flagging!